Makefile.am

```diff
@@ -90,13 +90,18 @@ endif # end ENABLE_RUST
 libglnx_srcpath := $(srcdir)/libglnx
 libglnx_cflags := $(OT_DEP_GIO_UNIX_CFLAGS) "-I$(libglnx_srcpath)"
 libglnx_libs := $(OT_DEP_GIO_UNIX_LIBS)
+$(srcdir)/libglnx/Makefile-libglnx.am.inc: $(srcdir)/libglnx/Makefile-libglnx.am
```

Comment:
Ah...wait, so you're only fixing a bug where you're using `./autogen.sh && make dist` and then building the resulting tarball?   This can't work for an `autogen.sh && make` build as we'll do the `include` *before* handling the rule, correct?

Reply:
(Also I pushed a fixup to add some comments)